src/uct/cuda/gdr_copy/gdr_copy_md.c

```diff
@@ -51,6 +52,10 @@ static ucs_config_field_t uct_gdr_copy_md_config_table[] = {
     {"MEM_REG_GROWTH", "0.06ns", "Memory registration growth rate", /* TODO take default from device */
      ucs_offsetof(uct_gdr_copy_md_config_t, uc_reg_cost.c), UCS_CONFIG_TYPE_TIME},
 
+    {"", "RCACHE_OVERHEAD=0;RCACHE_PURGE_ON_FORK=n;", NULL,
```

Comment:
1. is it really 0 overhead by default, or should it be some small number?
2. can we remove the last ; ?